src/main/java/com/puppycrawl/tools/checkstyle/api/JavadocTokenTypes.java

```diff
@@ -2567,7 +2567,34 @@ public final class JavadocTokenTypes {
      */
     public static final int INPUT_TAG = JavadocParser.RULE_inputTag + RULE_TYPES_OFFSET;
 
-    /** Isindex html tag. */
+    /**
+     * Isindex tag name.
+     *
+     * <p><b>Example:</b></p>
+     * <pre>{@code <isindex prompt="search"> }</pre>
```

Comment:
please wrap it in a head tag
https://www.tutorialspoint.com/html/html_isindex_tag.htm